app/models/project_step.rb

```diff
@@ -53,6 +59,7 @@ class NoChildrenAllowedError < StandardError; end
 
   before_save :handle_original_date_logic
   before_save :handle_finalized_at
+  before_save :handle_schedule_decendants
```

Comment:
You mean leave `scheduled_start_date` blank on steps with schedule_ancestors? I think that is preferable. Fuzzy says you all discussed this? Or are we talking about something different?